cpp/src/parquet/encoding.cc

```diff
@@ -2855,7 +2963,7 @@ class RleBooleanDecoder : public DecoderImpl, virtual public BooleanDecoder {
                              " (corrupt data page?)");
     }
     // Load the first 4 bytes in little-endian, which indicates the length
-    num_bytes = ::arrow::bit_util::ToLittleEndian(SafeLoadAs<uint32_t>(data));
```

Comment:
I think previously here could be strange, it should be `FromLittleEndian`

Reply:
Good catch

Reply:
(In fact, in little endian machine, they are just nop, and in big endian machine, they just do byteswap, but the syntax here is ambigious)